File: 0014-longest-common-prefix/0014-longest-common-prefix.cpp
class Solution {
public:
string longestCommonPrefix(vector<string> &strs)
{
    sort(strs.begin(), strs.end());
    string ans = "";
    int n = strs.size();
    for (int i = 0; i < strs[0].size(); i++)
    {
        if (strs[0][i] == strs[n - 1][i])
            ans += strs[0][i];
        else
            break;
    }
    return ans;
}
};

File: 0014-longest-common-prefix/0014-longest-common-prefix.java
class Solution {
    public String longestCommonPrefix(String[] strs) {
        String ans="";
        Arrays.sort(strs);
        int n=strs.length;
        for(int i=0;i<strs[0].length();i++){
            if(strs[0].charAt(i)==strs[n-1].charAt(i)){
                ans+=strs[0].charAt(i);
            }
            else
                break;
        }
        return ans;
    }
}